src/test/java/com/softserveinc/edu/ita/page_object/AdministrationPage.java

```diff
@@ -1,10 +1,58 @@
 package com.softserveinc.edu.ita.page_object;
 
+import com.softserveinc.edu.ita.locators.AdministrationPageLocators;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Random;
 
 public class AdministrationPage extends LogOutBase {
 
     public AdministrationPage(WebDriver driver) {
         super(driver);
     }
+
+    public NewUserPage clickCreateUserLink() {
+        driver.findElement(AdministrationPageLocators.CREATE_NEW_USER_LINK.getBy()).click();
+        return new NewUserPage(driver);
+    }
+
+    public void acceptAlert() {
```

Comment:
Protest. We talked to move if someone needs it, i've asked people in the morning, nobody needs.